app/microservice_tables.py

```diff
@@ -68,11 +75,38 @@ def load_cookie_into_fields(field_classes, previous_criteria):
       elif field.search_type == "dropdown":
         field.refresh_selected_dropdown(
             previous_criteria.get(field.database_name))
+      elif field.search_type == "checkbox":
+        field.refresh_dropdown_status(
+            previous_criteria.get(field.database_name))
+
   return field_classes
 
 
 def get_fields(service_name):
   # Set default Dropdown Values
+  job_role_dropdow_options = [
+      "Census area manager",
+      "Community adviser working with the Arab community",
+      "Community adviser working with the Turkish and Kurdish communities",
+      "Community adviser working with the Chinese community",
+      "Communal establishments team leader", "Communal establishments officer",
+      "Community adviser working with the Black Caribbean community",
+      "Census area manager (Welsh speaking)",
+      "Community adviser working with the Bangladeshi community",
+      "Census team leader",
+      "Community adviser working with the Pakistani community",
+      "Communal establishments officer (Welsh speaking)",
+      "Communal establishments area support", "Census engagement manager",
+      "Communal establishments area manager",
+      "Community adviser working with the Somali community",
+      "Community adviser working with the Nepali community",
+      "Community adviser working with the Black African community",
+      "Community adviser working with the Indian community",
+      "Census mobile team leader", "Census officer - 1st intake",
+      "Census area support", "Census engagement manager (Welsh speaking)"
+  ]
```

Comment:
Is hardcoding these okay?

Reply:
Hardcoding them for now will reduce the latency from having to get the values from the database, a caching solution is on another ticket for the future